### DISCUSSION TITLE
Add or delete rows in a table

### DISCUSSION THREAD

**Discussion by mmannj** [ocornut/imgui#5622] [New Users Build/Link/Run issues ONLY!]
Is there a way to add or delete a row in a table? Or make the row 'Hidden'?

**Comment by rokups** [comment]
Dear ImGui is immediate mode library and as such it renders only what you submit for it to render each frame. If you want to not render some rows in the table - then do not submit them. Read more about [immediate mode paradigm](https://github.com/ocornut/imgui/wiki/About-the-IMGUI-paradigm).
